docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-linux.md

```diff
@@ -10,9 +10,194 @@ If you do not have a DirectX 11 capable GPU in your system (for example if you a
+
+ sudo dnf config-manager \
+    --add-repo \
+    https://download.docker.com/linux/fedora/docker-ce.repo
+```
+
+Install the latest version of docker engine, containerd, and docker compose
+```
+sudo dnf install docker-ce docker-ce-cli containerd.io docker-compose-plugin
+```
+If prompted to accept the GPG key, verify that the fingerprint matches `060A 61C5 1B55 8A7F 742B 77AA C52F EB6B 621E 9F35` (cross reference this with the [GPG key fingerprint provided by dockers docs](https://docs.docker.com/engine/install/fedora/)) , and if so, accept it.
+
+Start docker
+```
+sudo systemctl start docker
+```
+
+Then verify that docker is installed correctly by running this
+```
+sudo docker run hello-world
+```
+
+### Prep Titanfall Server files
+
+You will need obtain Titanfall's game files, typically by copying over your Titanfall installation folder and placing it on the Linux machine. Most of these files are for SP and therefore we can delete from the installation folder and prune the file size down to \~5GB.
+
+Easiest way to do this is to copy the entire Titanfall2 folder to the server and delete the [following files](https://github.com/pg9182/northstar-dedicated#reducing-the-size-), or alternatively download and run [this sh script](https://gist.github.com/pg9182/9a962adbfc27e93237cd14e4523c9da8) made by pg9182 (note: this only downloads assets necessary to run a server and does not download a copy of the game).
+
+#### Copy Titanfall Folder over to the Linux Machine
+
+1. Access the files on your Linux machine using tools like [WinSCP](https://winscp.net/eng/download.php)
+2. Navigate to the folder where you want to store the files. You can put them into `~/Titanfall2` for example.
+3. Copy the newly pruned Titanfall folder to the server.
+
+![Game files copied to `/mnt/Titanfall` as an example](https://i.postimg.cc/15HbbzFr/image.png)
+
+#### Copy mods to server
+
+If you have configured some mods,these can be placed at a similar location, like `~/mods` for example
+![Mods placed inside /mnt/mods](https://i.postimg.cc/tRD5jnrJ/image.png)
+
+#### Create docker-compose file
+
+We are going to be using Docker-Compose to set up our container, this gives us much more flexibility and allows us to make changes to the start up args much cleaner. This example will let you start your servers without needing to have your compose in the same folder and directory as Titanfall 2 and your mods folder. Create a compose file in your home directory, or wherever is most convenient.
+
+```
+nano docker-compose.yml
+```
+
+Example docker-compose.yml
+
+```yaml
+version: '3'
+services:
+ northstar-skirmish: 
+   image: ghcr.io/pg9182/northstar-dedicated:1-tf2.0.11.0 
+   pull_policy: always 
+   environment:
+     - NS_PORT=37015
+     - NS_PORT_AUTH=8081
+     - 'NS_SERVER_NAME=Enter Server Name here'
+     - 'NS_SERVER_DESC=Enter your description here'
+     - NS_EXTRA_ARGUMENTS=
+       +setplaylist aitdm //skirmish
+       +mp_gamemode aitdm //skirmish
```

Comment:
YAML uses `#` for comments not `//` and `aitdm` is Attrition, not Skirmish :P

Reply:
i forgor :skull:

Reply:
fixed instances of `skirmish` and `//` being used (whorps)